### PR TITLE
Rename /tvinna to /jobs/tvinna, allowing for more future job listings

### PR DESCRIPTION
**Pull request by benediktvaldez**
Just a thought! Applying the same logic as with /firm/is for icelandic companies

### CODE REVIEW

File: endpoints/jobs/test.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 var endpoint = require('./');
 
-describe('Searching', function() {
+describe('Tvinna', function() {
```
Comment (koddsson) [inline endpoints/jobs/test.js]: Should this now be:

```javascript
describe('Jobs', function() {
```

and that could wrap the old test?

**Comment by koddsson** [comment]
LGTM

**Comment by kristjanmik** [comment]
Shouldn't we add the /is if we are following the same logic as the firm endpoint? Tvinna is just providing jobs in iceland right? Sustaining this logic might get complicated since some things are running in 2 countries

**Comment by benediktvaldez** [comment]
@kristjanmik only problem I see with that is that right now it's based on the source, in this example Tvinna.is, which theoretically could show job listings outside of Iceland

Later we could compile all jobs related to iceland specifically from all available sources and publish them under `/is`

**Pull request merged**